front/app/component-library/components/Button/Button.stories.tsx

```diff
@@ -35,20 +35,6 @@ export const PrimaryOutlined = {
   },
 };
 
-export const PrimaryInverse = {
-  args: {
-    buttonStyle: 'primary-inverse',
-    children: 'Button',
-  },
-};
```

Comment:
Why are we removing this one? We still have `primary-inverse` in use, right?

Reply:
Although the name is strange, since it doesn't use the same colors as `primary`.

Reply:
Yes, this was a mistake, I'll bring it back 👍